packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.test.ts

```diff
@@ -23,272 +24,320 @@ describe('DateMath', () => {
```

Comment:
>.to.be.true; [](start = 53, length = 12)

mentione offline, this needs to be .toBe(true);

Reply:
have it as toEqual()